src/shared/components/card/card.tsx

```diff
@@ -0,0 +1,34 @@
+import type { MouseEvent } from 'react';
+
+import * as styles from './card.css';
+
+interface CardProps {
+  title: string;
+  major?: string;
+  description: string;
+  img: string;
+  type: 'details' | 'apply';
```

Comment:
type이라는 props명은 너무 직관적이지 않을것 같아요!
사이즈를 나타낼거면 sm, lg를 사용하는게 더 직관적일것 같다는 생각입니다!
```suggestion
  type: 'sm' | 'lg';
```